covid_19/covid19_and_era5_matching.ipynb

```diff
@@ -0,0 +1,7632 @@
+{
```

Comment:
<p>Can  you add a bit of documentation that outlines the steps in this notebook. Just a narrative description of what we're doing and where the data is coming from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/Pangeo_KONQUER/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
You want this as a Markdown cell at the beginning of the notebook?

Reply:
That's what I was thinking.

Reply:
Added!